src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1065,11 +1065,15 @@ def config_diagnostics(resource_group_name, name, level=None,
         application_logs = ApplicationLogsConfig(fs_log)
 
     http_logs = None
-    if web_server_logging is not None:
-        enabled = web_server_logging
-        # 100 mb max log size, retenting last 3 days. Yes we hard code it, portal does too
-        fs_server_log = FileSystemHttpLogsConfig(100, 3, enabled)
-        http_logs = HttpLogsConfig(fs_server_log)
+    server_logging_option = web_server_logging or docker_container_logging
+    if  server_logging_option:
+        # TODO: az blob storage log config currently not in use, will be impelemented later
```

Comment:
I suggest instead of adding `TODO` comments, which are usually ignored, please open new CI issue to track this work item.

Reply:
Okay I've opened a new issue here -> #4764 Although I would like to keep the `TODO` here as well.

Reply:
Can you add the issue number to the comment so that two things are associated.